### ISSUE TITLE
Доступ к любому файлу на сервере.

### ISSUE THREAD

**Issue by vgrabko** [splincode/apachengine#2]
Через вашу панельку можно получить доступ к любому файлу. Надо со всех переменных удалять последовательности ../ дабы хакер не мог выйди в другие директории.

**Comment by splincode** [comment]
Да, об этом я упомянул в примечании, скорее всего я могу сделать консольную программку, но смысл в ней. Я хотел получить нечто похожее на denwer в Windows.

**Comment by vgrabko** [comment]
Так работы на пол часа для исправления.

**Comment by splincode** [comment]
То есть? Можно исправить вариант с доступом к любому файлу? Покажите, внесите изменения в код
